### PR TITLE
az-logout: add German translation

### PR DESCRIPTION
**Pull request by sidyes**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.de/common/az-logout.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# az logout
+
+> Melden Sie sich ab, um den Zugriff auf Azure-Abonnements zu entfernen.
+> Teil der `Azure CLI`, der Befehlszeilenschnittstelle von Azure.
+> Mehr Informationen: <https://learn.microsoft.com/cli/azure/reference-index#az_logout>.
```
Comment (kbdharun) [inline pages.de/common/az-logout.md:5]:
```suggestion
> Weitere Informationen: <https://learn.microsoft.com/cli/azure/reference-index#az_logout>.
```

**Review by kbdharun — APPROVED** [review]

File: pages.de/common/az-logout.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# az logout
+
+> Melden Sie sich ab, um den Zugriff auf Azure-Abonnements zu entfernen.
+> Teil der `Azure CLI`, der Befehlszeilenschnittstelle von Azure.
```
Comment (pixelcmtd) [inline pages.de/common/az-logout.md:4]:
```suggestion
> Teil von `azure-cli`.
```

File: pages.de/common/az-logout.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# az logout
+
+> Melden Sie sich ab, um den Zugriff auf Azure-Abonnements zu entfernen.
```
Comment (pixelcmtd) [inline pages.de/common/az-logout.md:3]:
```suggestion
> Von Azure-Abonnements abmelden.
```

File: pages.de/common/az-logout.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+> Teil der `Azure CLI`, der Befehlszeilenschnittstelle von Azure.
+> Weitere Informationen: <https://learn.microsoft.com/cli/azure/reference-index#az_logout>.
+
+- Melden Sie das aktuelle aktive Konto ab:
```
Comment (pixelcmtd) [inline pages.de/common/az-logout.md:7]:
```suggestion
- Melde das aktuelle aktive Konto ab:
```

File: pages.de/common/az-logout.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+`az logout`
+
+- Melden Sie einen spezifischen Kontobenutzer ab:
```
Comment (pixelcmtd) [inline pages.de/common/az-logout.md:11]:
```suggestion
- Melde einen spezifischen Benutzernamen ab:
```

File: pages.de/common/az-logout.md (hunk 2 above)
Comment (pixelcmtd) [reply pages.de/common/az-logout.md:4]: @sidyes What exactly is your rationale for marking this as resolved? You merged basically the same suggestion in #9171, so please at least explain why you disagree here.

Comment (sidyes) [reply pages.de/common/az-logout.md]: Sorry that was a mistake

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**